examples/infinite-scroll/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "example-infinite-scroll",
+  "private": true,
+  "prettier": "@shopify/prettier-config",
+  "scripts": {
+    "build": "shopify hydrogen build --diff",
+    "dev": "shopify hydrogen dev --codegen --diff",
+    "preview": "npm run build && shopify hydrogen preview",
+    "lint": "eslint --no-error-on-unmatched-pattern --ext .js,.ts,.jsx,.tsx .",
+    "codegen": "shopify hydrogen codegen"
+  },
+  "dependencies": {
+    "@remix-run/react": "^2.5.1",
+    "@remix-run/server-runtime": "^2.5.1",
+    "@shopify/cli": "3.52.0",
+    "@shopify/cli-hydrogen": "^7.0.0",
+    "@shopify/hydrogen": "~2024.1.0",
+    "@shopify/remix-oxygen": "^2.0.3",
+    "@total-typescript/ts-reset": "^0.4.2",
+    "graphql": "^16.6.0",
+    "graphql-tag": "^2.12.6",
+    "isbot": "^3.6.6",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
```

Comment:
We can remove these since they are merged from the skeleton dependencies.

```suggestion
```

Reply:
Are you saying it's smart enough for me to only put in `react-intersection-observer`, and it will merge everything else? 🤯